yoti/YotiWidget.php

```diff
@@ -24,14 +27,27 @@ public function __construct()
      */
     public function widget($args, $instance)
     {
+        if ( ! isset( $args['widget_id'] ) ) {
+            $args['widget_id'] = $this->id;
+        }
+        $title = (!empty( $instance['title'])) ? $instance['title'] : __(self::YOTI_WIDGET_DEFAULT_TITLE);
+
+        $title = apply_filters('widget_title', $title, $instance, $this->id_base);
+
         wp_enqueue_style('yoti-asset-css', plugin_dir_url(__FILE__) . 'assets/styles.css');
         $config = YotiHelper::getConfig();
-        if (!empty($config['yoti_sdk_id']) && !empty($config['yoti_pem']['contents'])) {
-            echo '<div class="yoti-connect-button">' . YotiButton::render(NULL, TRUE) . '</div>';
+        $widgetTitleHtml = '';
+        $widgetContent = '<strong>Yoti not configured.</strong>';
+        // Apply widget title html
+        if(!empty($title)){
+            $widgetTitleHtml = $args['before_title'] . $title . $args['after_title'];
         }
-        else {
-            echo '<div class="yoti-missing-config"><p><strong>Yoti Connect not configured.</strong></p></div>';
+        if (!empty($config['yoti_sdk_id']) && !empty($config['yoti_pem']['contents'])) {
+            $widgetContent = YotiButton::render(NULL, TRUE);
         }
+        echo $args['before_widget'];
+        echo $widgetTitleHtml . "<ul><li>$widgetContent</li></ul>";
```

Comment:
Why use `<ul><li>` here? it will add bullets on some themes.

Reply:
I am following the widget structure as you requested. Drupal is applying the widget style following that structure. You need `ul, li` in there. And I wouldn't worry about that as WP is taking care of removing the bullet.
e.g 
.widget ul {
    list-style: none;
    margin: 0;
}

Reply:
That CSS rule is part of the theme. I think we have to add it to the Yoti plugin to make sure WordPress doesn't add the bullet.

Reply:
Ok I will add it in Yoti stylesheet

Reply:
I have added the Widget style to Yoti button style.